avocado/plugins/spawners/podman.py

```diff
@@ -93,25 +97,51 @@ def is_task_alive(self, runtime_task):
         # FIXME: check how podman 2.x is reporting valid "OK" states
         return out.startswith(b'Up ')
 
+    def _fetch_asset(self, url):
+        cachedirs = self.config.get('datadir.paths.cache_dirs')
+        asset = Asset(url, cache_dirs=cachedirs)
+        return asset.fetch()
+
+    def get_needed_eggs(self, py_major, py_minor):
+        """Return the basic eggs needed to bootstrap Avocado."""
+        result = []
+        # Setuptools
+        # For now let's pin to setuptools 59.2.
+        # TODO: Automatically get latest setuptools version.
+        eggs = [f"https://github.com/avocado-framework/setuptools/releases/download/v59.2.0/setuptools-59.2.0-py{py_major}.{py_minor}.egg",
+                f"https://github.com/avocado-framework/avocado/releases/download/{VERSION}/avocado_framework-{VERSION}-py{py_major}.{py_minor}.egg"]
+        for url in eggs:
+            path = self._fetch_asset(url)
+            to = os.path.join('/tmp/', os.path.basename(path))
```

Comment:
I understood this `to` variable just after reading below. This is the target directory inside the container where the eggs will be copied to.

Personally, I would not expect this method to decide and define this location. The name `get_needed_eggs`, as I understand, should return a list of where, in the cache, the eggs are. Later you can define the target directory inside the container.

Reply:
I understand your point. Since this method is used in two steps of the process (before the container start and after), I decided to go this way. But yes your comment makes sense. I will rename the method to make it clear.